geomstats/_backend/pytorch/random.py

```diff
@@ -16,6 +16,8 @@ def seed(*args, **kwargs):
 
 
 def normal(loc=0.0, scale=1.0, size=(1,)):
+    if isinstance(size, int):
```

Comment:
I think it's better to test whether `size` is not a sequence by doing `if not hasattr(size, '__iter__')`. Keep in mind that a numpy integer like `np.int32(1)` does not inherit from python's builtin `int` type.

Edit: I also realize that the same logic should apply to `uniform` below, which should have `size=(1,)` as default, and then check whether the `size` parameter is iterable.

Reply:
Smart ! I'll change that.